libs/ledger-live-common/src/families/solana/setup.ts

```diff
@@ -92,6 +99,11 @@ const createSigner: CreateSigner<SolanaSigner> = (transport: Transport) => {
 
       return app.signTransaction(path, tx);
     },
+    signMessage: (path: string, message: string) => {
+      return app
+        .signOffchainMessage(path, toOffChainMessage(message))
+        .then(response => ({ signature: Buffer.from(response.signature.toString("hex")) }));
```

Comment:
You probably need to specify the encoding if you want to go to `hex`
```suggestion
        .then(response => ({ signature: Buffer.from(response.signature.toString("hex"), "hex") }));
```

But I'm not sure we actually want this
It's probably better to just keep the buffer in `response.signature` as is and only change it in `libs/coin-modules/coin-solana/src/hw-signMessage.ts`

Reply:
done